client/my-sites/site-settings/form-seo.jsx

```diff
@@ -204,6 +207,7 @@ export const SeoForm = React.createClass( {
 
 		const isSitePrivate = parseInt( blog_public, 10 ) !== 1;
 		const isDisabled = isSitePrivate || isSubmittingForm;
+		const isSaveDisabled = isDisabled || isSubmittingForm || ( ! showPasteError && invalidCodes.length > 0 );
```

Comment:
I mentioned this in another issue https://github.com/Automattic/wp-calypso/issues/5612 but the entire form should be disabled while the settings are loading. This is generally done with a `disabled` attribute on every field.